cookbooks/swift/templates/default/etc/swift/object-server/server.conf-template.erb

```diff
@@ -10,16 +10,7 @@ pipeline = <%= @zipkin %> recon object-server
 
 [app:object-server]
 use = egg:swift#object
+replication_server = <%= @replication_server %>
```

Comment:
Maybe I should make this more like
```
<% if ! @replication_server -%>
replication_server = False
<% end -%>
```
so it mostly works with pre-[740867](https://review.opendev.org/#/c/740867/) swift...